common/djangoapps/philu_commands/helpers.py

```diff
@@ -26,7 +26,7 @@ def generate_course_structure(course_key):
 
             if (curr_block.category == 'discussion' and
                 hasattr(curr_block, 'discussion_id') and
```

Comment:
Can we format this better as well?

Reply:
@MunamMubashir-arbisoft Can you please have one final look at this change? I have updated this condition. Just have a look at the latest commit.